frontend/src/util/DateUtils.ts

```diff
@@ -36,11 +36,23 @@ export function useDateUtils() {
         });
     }
 
+    /**
+     * es muss für i18n ein Datumsobjekt erzeugt werden.
+     */
+    function getDatumOfString(datum: string): Date {
+        const d = datum;
+        if (Date.parse(d)) {
+            return new Date(d);
+        }
```

Comment:
fehler in der Snackbar ausgeben

Reply:
War noch nie drinnen, ist aber wahrscheinlich nicht verkehrt